### PR TITLE
ls: add Japanese translation

### PR DESCRIPTION
**Pull request by shu-pf**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This is my first time participating in OSS.
I would appreciate it if you could point out any inaccuracies.

### CODE REVIEW

**Comment by marchersimon** [comment]
If you're working with git, here's how to do that: https://github.com/microsoft/vscode/issues/116341#issue-805637206

**Comment by shu-pf** [comment]
@marchersimon 
Thank you for letting me know.
I have changed the default branch in the fork.

File: pages.ja/common/ls.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- サイズを人間が読みやすい単位（KiB、MiB、GiB）で表示した長い形式での一覧表示:
+
+`ls -lh`
+
+- サイズ順（降順）に並べた長い形式での一覧表示:
+
+`ls -lS`
+
+- すべてのファイルの長い形式でのリストで、更新日が古いものから順に表示されます。:
```
Comment (bl-ue) [inline pages.ja/common/ls.md:30]: A period and a colon here?
```suggestion
- すべてのファイルの長い形式でのリストで、更新日が古いものから順に表示されます:
```

**Review by shu-pf — COMMENTED** [review]
Removed punctuation.

File: pages.ja/common/ls.md (hunk 1 above)
Comment (shu-pf) [reply pages.ja/common/ls.md]: No that is my mistake.
I appreciate you pointing it out.

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
Thank you for folowing the contributing guidelines so well! :)

Welcome to tldr-pages!

**Pull request merged**